sw/device/tests/usbdev_mixed_test.c

```diff
@@ -46,7 +46,7 @@
 #define NUM_STREAMS USBUTILS_STREAMS_MAX
 #endif
 
-#define TRANSFER_BYTES_SILICON (0x10U << 20)
+#define TRANSFER_BYTES_SILICON (0x10U << 16)
```

Comment:
I see the previous 16MiB amount timing out on Si, but 4MiB takes about 20 seconds to complete.
usbdev_iso_test and usbdev_mixed_test should be about the same as each other, being limited by the 64 bytes/1ms polling rater of Isochronous and Interrupt endpoints.